.ci/travis/ubuntu/Dockerfile

```diff
@@ -11,11 +11,13 @@ RUN apt-get update && apt-get install -y \
   clang-format-3.9 \
   git \
   curl \
-  protobuf-compiler \
-  libprotoc-dev \
   zlib1g \
   zlib1g-dev \
-  libtinfo-dev && \
+  libtinfo-dev \
+  unzip \
+  autoconf \
```

Comment:
Please add information about these new requirements to the docs.

Reply:
Done